Description: Fix line break in generate man page to make lintian happy.
Author: Andrew Starr-Bochicchio <a.starr.b@gmail.com>
Bug: https://bugs.launchpad.net/libgtk2-appindicator-perl/+bug/1033655
Forwarded: yes

Index: unstable/lib/Gtk2/AppIndicator.pm
===================================================================
--- unstable.orig/lib/Gtk2/AppIndicator.pm	2012-11-18 11:55:31.924164000 -0500
+++ unstable/lib/Gtk2/AppIndicator.pm	2012-11-18 11:58:51.054866369 -0500
@@ -284,7 +284,7 @@
  $ind=Gtk2::AppIndicator->new($application_id,$active_icon_name [,$category])
  
 Creates a new application indicator object with given name (id) and icon name for the active icon.
-Category must be one of  { 'application-status','communications','system-services','hardware','other' }
+Category must be one of  { 'application-status', 'communications', 'system-services', 'hardware', 'other' }
 if set. if not set, it defaults to 'application-status'.
 
  $ind->set_icon_theme_path($path)
